modules/@apostrophecms/import-export-page/index.js

```diff
@@ -74,7 +74,11 @@ module.exports = {
           return self.apos.modules['@apostrophecms/job'].run(
             req,
             (req, reporting) => self.apos.modules['@apostrophecms/import-export']
-              .export(req, self, reporting)
+              .export(req, self, reporting),
+            {
+              action: 'export',
+              docTypes: []
```

Comment:
This way, the export batch action will never trigger a manager or editor refresh which makes sense.